src/plugins/intel_cpu/src/nodes/mkldnn_conv_node.h

```diff
@@ -92,16 +87,17 @@ class MKLDNNConvolutionNode : public MKLDNNNode {
     bool isPossibleToSkipInitConfig(MKLDNNDescriptor &desc) const;
     bool isNspcAvailable() const;
     InferenceEngine::Blob::Ptr createInternalBlob(InferenceEngine::SizeVector dims, size_t edgeNum, bool isGrouped = false);
-    std::shared_ptr<mkldnn::convolution_forward::desc>
-    createDescriptorInternal(const mkldnn::memory::desc& inputDesc,
-                             const mkldnn::memory::desc& weightDesc,
-                             const mkldnn::memory::desc& outputDesc,
-                             mkldnn::algorithm alg);
-    std::shared_ptr<mkldnn::convolution_forward::desc>
+
+    static std::shared_ptr<mkldnn::convolution_forward::desc>
     createDescriptorInternal(const mkldnn::memory::desc& inputDesc,
                              const mkldnn::memory::desc& weightDesc,
                              const mkldnn::memory::desc& biasDesc,
                              const mkldnn::memory::desc& outputDesc,
+                             bool withBiases,
+                             const std::vector<size_t>& stride,
+                             const std::vector<ptrdiff_t>& dilation,
+                             const std::vector<ptrdiff_t>& paddingL,
+                             const std::vector<ptrdiff_t>& paddingR,
                              mkldnn::algorithm alg);
```

Comment:
Is not it better to hide this function in the cpp? Because it is not a method and I doubt it will be called somewhere else.